python/lsst/obs/rubinGenericCamera/rawFormatter.py

```diff
@@ -0,0 +1,76 @@
+__all__ = ["StarTrackerNarrwRawFormatter", "StarTrackerWideRawFormatter", "StarTrackerFastRawFormatter",]
+
+from .translator import StarTrackerNarrwTranslator, StarTrackerWideTranslator, StarTrackerFastTranslator
+from lsst.obs.base import FitsRawFormatterBase
+from .filters import RUBIN_GENERIC_CAMERA_FILTER_DEFINITIONS
+from ._instrument import StarTrackerNarrw, StarTrackerWide, StarTrackerFast
+from lsst.obs.base import InitialSkyWcsError
+
+
+class RubinGenericCameraRawFormatter(FitsRawFormatterBase):
+    cameraClass = None
+    translatorClass = None
+    filterDefinitions = RUBIN_GENERIC_CAMERA_FILTER_DEFINITIONS
+
+    def getDetector(self, id):
+        return self.cameraClass().getCamera()[id]
+
+    def makeWcs(self, visitInfo, detector):
```

Comment:
I'm trying to work out how this differs from the base class implementation.

Reply:
Good question.  Experimentally, we need `getDetector` but not `makeWcs`